modules/@apostrophecms/area/ui/apos/components/AposAreaMenuItem.vue

```diff
@@ -42,6 +42,7 @@ export default {
   methods: {
     click() {
       this.$emit('click');
+      this.$parent.close();
```

Comment:
Please add a comment for this that it's triggering the `AposContextMenu` method. I was thrown by this since there are so many components in play and using slots.

Reply:
commented about slots not emitting events